src/util/storagePool.tsx

```diff
@@ -59,3 +72,15 @@ export const testDuplicateStoragePoolName = (
     },
   ];
 };
+
+export const isPowerflexIncomplete = (
+  formik: FormikProps<StoragePoolFormValues>,
+): boolean => {
+  return (
+    formik.values.driver === powerFlex &&
+    (!formik.values.powerflex_domain ||
```

Comment:
since powerflex_domain is not required unless if powerflex_pool is set, do we try changing the logic here? Although it could be tricky since powerflex_domain is only reuiquired if powerflex_pool is the name and not id of the storage pool

Reply:
Well spotted! I'll relax the condition here to make the domain not required. If it is required, the backend will tell with an error anyways.